File: Examples/FormFieldNamesExample.cs
using ceTe.DynamicPDF.Forms;
using ceTe.DynamicPDF.Merger;
using System;

namespace DynamicPDFCoreSuite.Examples
{
    class FormFieldNamesExample
    {
        public static void Run()
        {
            ceTe.DynamicPDF.Merger.MergeDocument document = new MergeDocument(Util.GetPath("Resources/PDFs/AllFormFields.pdf"));

            FormFieldList list = document.Form.Fields;

            for(int i = 0; i < list.Count;i++)
            {
                Console.WriteLine(list[i].FullName);
            }

        }
    }
}

File: dynamicpdf-vbnet-examples/Examples/FormFieldNamesExample.vb
Imports ceTe.DynamicPDF.Forms
Imports ceTe.DynamicPDF.Merger

Class FormFieldNamesExample

    Public Shared Sub Run()

        Dim document As New MergeDocument(Util.GetPath("Resources/PDFs/AllFormFields.pdf"))
        Dim list As FormFieldList = document.Form.Fields
        For i As Integer = 0 To list.Count - 1
            Console.WriteLine(list(i).FullName)
        Next
    End Sub
End Class
